client/src/components/QandA/QandAContainer.jsx

```diff
@@ -90,7 +91,6 @@ function QandA({ product }) {
 
   const handleMoreAnsweredQuestions = () => {
     const howMany = questions.length - questionCount;
-    console.log(howMany);
     switch (howMany) {
```

Comment:
Since this switch statement only has one case, try refactoring this using a conditional instead